src/com/lilithsthrone/game/character/body/Body.java

```diff
@@ -1499,10 +1499,13 @@ public static Body loadFromXML(StringBuilder log, Element parentElement, Documen
 			}
 			
 			AbstractBreastType crotchBoobType = BreastType.getBreastTypeFromId(breasts.getAttribute("type"));
-			if(Main.isVersionOlderThan(Game.loadingVersion, "0.3.0.6")
+			
+			// disabled because it was erroneously applying crotch boobs to every non-human on load
+			/*if(Main.isVersionOlderThan(Game.loadingVersion, "0.3.0.6")
```

Comment:
I think I've fixed this on my end already, by changing it to:
`if(Main.isVersionOlderThan(Game.loadingVersion, "0.3.0.6") && importedLeg.getLegConfiguration().isBipedalPositionedCrotchBoobs()) { // Reset crotch-boob type as I accidentally applied crotch-boobs to demons
	if(body.isFeminine()) {
		crotchBoobType = RacialBody.valueOfRace(body.getRace()).getBreastCrotchType();
	} else {
		crotchBoobType = BreastType.NONE;
	}
}`